src/core/hle/Patches.cpp

```diff
@@ -438,8 +438,14 @@ inline void EmuInstallPatch(const std::string FunctionName, const xbox::addr_xt
         return;
     }
 
-	g_FunctionHooks[FunctionName].Install((void*)(FunctionAddr), (void*)patch.patchFunc);
-	printf("HLE: %s Patched\n", FunctionName.c_str());
+	auto result = g_FunctionHooks[FunctionName].Install((void*)(FunctionAddr), (void*)patch.patchFunc);
+	if (!result) {
+		printf("HLE: %s Patch Failed\n", FunctionName.c_str());
+	}
+	else {
+		printf("HLE: %s Patched\n", FunctionName.c_str());
+	}
```

Comment:
My advice to change this into...
```suggestion
	auto success = g_FunctionHooks[FunctionName].Install((void*)(FunctionAddr), (void*)patch.patchFunc);
	if (success) {
		printf("HLE: %s Patched\n", FunctionName.c_str());
	}
	else {
		printf("HLE: %s Patch Failed\n", FunctionName.c_str());
	}
```
This way we don't need `not` operator and to make the wording clear.